app/css/animator.css

```diff
@@ -2,8 +2,6 @@
 
 html {
   height: 100%;
-  min-width: 590px;
-  min-height: 590px;
```

Comment:
What was the purpose behind removing this? They prevent the layout from resizing beyond the minimum size utterly breaking.